Rubberduck.VBEEditor/SafeComWrappers/VB/Enums/VBEKind.cs

```diff
@@ -1,17 +1,15 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace Rubberduck.VBEditor.SafeComWrappers
+﻿namespace Rubberduck.VBEditor.SafeComWrappers
 {
     public enum VBEKind
     {
-        /// <summary>Embedded VB editor (Visual Basic for Applications).</summary>
-        Embedded,
+        /// <summary>
+        /// Hosted VB editor (Visual Basic for Applications).
+        /// </summary>
+        Hosted,
 
-        /// <summary>Standalone VB editor (Visual Basic).</summary>
+        /// <summary>
+        /// Standalone VB editor (Visual Basic).
+        /// </summary>
         Standalone
```

Comment:
FWIW, this might not correspond to the project kind enumeration in VBA which has same constant names. Not that it's likely we'll encounter a truly standalone VBA project since they no longer can be made since 2003 but it bears noting that the project kind enumeration `Standalone` won't be equivalent to this.

Reply:
AFAIK, VBProject.ProjectKind is now more or less useless, so this shouldn't be a problem for us.

Reply:
I agree it's useless now but because the names are same, I think xml-doc needs to mention that it has nothing to do with that enum.